### DISCUSSION TITLE
Incompatible Yarn Version

### DISCUSSION THREAD

**Discussion by Top-Dev-Ops** [vercel/community#737] [Help] [answered]
Hello guys, i am deploying a next.js app on Vercel but faced the following error.
`error autonomy@0.1.0: The engine "yarn" is incompatible with this module. Expected version ">=1.22.18". Got "1.22.17"`
Can anyone please help me how to set the yarn version in Vercel manually?
I have been spending several hours fixing this problem without any success.
Thanks in advance

**Comment by amyegan** [comment, marked as answer]
Hi @Top-Dev-Ops. If you want to use a specific version of yarn, you can use the experimental Corepack build config feature. https://vercel.com/docs/concepts/deployments/configure-a-build#corepack

**Reply by Top-Dev-Ops** [reply]
Big Thanks!
